roles/conjur_host_identity/tests/test.sh

```diff
@@ -71,7 +73,7 @@ function run_test_case {
     "
     docker exec "${ansible_cid}" bash -ec "
       cd tests
-      py.test --junitxml=./junit/${test_case} --connection docker -v test_cases/${test_case}/tests/test_default.py
+      py.test --junitxml=./junit/${test_case} --connection docker -v test_cases/common/test_default.py
```

Comment:
Initially my feedback was going to be. Let's add a comment up top to clarify why we avoid this folder, and a comment here to clarify that the common folder contains the code for the shared common test cases. Now I'm thinking that we should perhaps
1. Have common folder that has those common test cases in test_default.py
2. Retain the convention/interface of each test case needing to specify a `test_default.py` so that line 76 statys as
```
py.test --junitxml=./junit/${test_case} --connection docker -v test_cases/common/test_default.py
```
3. In each test case's `test_default.py` it can just import (1)